core/utils/appium_python.py

```diff
@@ -9,3 +9,8 @@ def __init__(self, capabilities):
         self.capabilities = capabilities
         self.executor = 'http://0.0.0.0:4723/wd/hub'
         self.driver = webdriver.Remote(self.executor, capabilities)
+
+    @staticmethod
+    def quit():
```

Comment:
I think this should be instance method that will destroy the appium server from the constructor instead of creating new instance

Reply:
Will add the change in separate PR.